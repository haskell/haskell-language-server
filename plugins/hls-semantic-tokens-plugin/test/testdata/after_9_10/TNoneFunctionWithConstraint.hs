module TNoneFunctionWithConstraint where

x :: (Eq a) => a
x = undefined

