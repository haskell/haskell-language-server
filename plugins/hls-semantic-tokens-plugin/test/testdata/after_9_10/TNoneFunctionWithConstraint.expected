1:8-35 TModule "TNoneFunctionWithConstraint"
3:1-2 TVariable "x"
3:7-9 TClass "Eq"
3:10-11 TTypeVariable "a"
3:16-17 TTypeVariable "a"
4:1-2 TVariable "x"
4:5-14 TVariable "undefined"
